runtime/lua/vim/treesitter/query.lua

```diff
@@ -34,14 +69,29 @@ local function safe_read(filename, read_quantifier)
   return content
 end
 
+--- Disables loading query files matching provided patterns
+---
+---@param lang string
+---@param patterns string[]
+function M.ignore_query_files(lang, patterns)
+  if not query_file_config.ignore_patterns[lang] then
+    query_file_config.ignore_patterns[lang] = {}
+    query_file_config.ignored_files[lang] = {}
+  end
```

Comment:
Following @gpanders comment, I would do the following instead:
```suggestion
  query_file_config.ignore_patterns[lang] = {}
  query_file_config.ignored_files[lang] = {}
```